neuraxle/steps/flow.py

```diff
@@ -0,0 +1,43 @@
+from neuraxle.base import MetaStepMixin, BaseStep, ExecutionContext, DataContainer, NonTransformableMixin, \
+    ExecutionMode, NonFittableMixin
+
+
+class TransformOnlyWrapper(NonTransformableMixin, NonFittableMixin, MetaStepMixin, BaseStep):
```

Comment:
Do all those wrappers also work on regular steps (e.g.: a `Normalizer()`), or is it just for checkpoints? 
It'd be freakin awesome to be able to process all step types. Otherwise, move this file to a new checkpoint package.

Reply:
yes it works with any kind of step @guillaume-chevalier

Reply:
Cool, does it works like the OutputTransformerMixin? This mixin could be in the same file as the `TransformOnlyWrapper` and friends. I believe they work in a similar way.